components/FooterBar/FooterBar.js

```diff
@@ -0,0 +1,44 @@
+// import Link from 'components/Link'
+// import { FOOTER_LINKS } from './config'
+import { EyeIcon } from '@heroicons/react/solid'
+import Link from 'components/Link'
+
+const SOCIALS = [
+  {
+    to: 'https://www.facebook.com',
+    title: 'Facebook icon'
+  },
+  {
+    to: 'https://www.twitter.com',
+    title: 'Twitter icon'
+  },
+  {
+    to: 'https://www.snapchat.com',
+    title: 'Snapchat icon'
+  },
+  {
+    to: 'https://www.instagram.com',
+    title: 'Instagram icon'
+  }
+]
+
+const FooterBar = () => (
```

Comment:
Could be simplified to `Footer`

- Bit predantic but overal simplier and just as sematic.